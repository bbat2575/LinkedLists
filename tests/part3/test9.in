NEW 1
1
NEW 2
ha
bazinga
NEW 3
@
bee
c
NEW 7
no
{0}
&
{1}
then
{2}
!
DELETE 3 1
DELETE 3 -2
DELETE 3 2
INSERT 3 -4 {0}
INSERT 3 -2 {2}
INSERT 3 3 {1}
DELETE 3 -6
DELETE 3 4
DELETE 3 -3
