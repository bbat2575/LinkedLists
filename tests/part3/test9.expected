
Type HELP for a list of options
Press <CTRL + D> to Quit

> List 0: 1

> List 1: ha -> bazinga

> List 2: @ -> bee -> c

> Nested 3: no -> {List 0} -> & -> {List 1} -> then -> {List 2} -> !

> List 3: no -> & -> {List 1} -> then -> {List 2} -> !

> List 3: no -> & -> {List 1} -> then -> !

> List 3: no -> & -> then -> !

> Nested 3: no -> {List 0} -> & -> then -> !

> Nested 3: no -> {List 0} -> & -> then -> {List 2} -> !

> Nested 3: no -> {List 0} -> & -> {List 1} -> then -> {List 2} -> !

> List 3: no -> & -> {List 1} -> then -> {List 2} -> !

> List 3: no -> & -> {List 1} -> then -> !

> List 3: no -> & -> then -> !

> 
